visual-js/visual-wdio/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:18 AS runner
```

Comment:
Shouldn't we mainly test on the latest LTS ?

Reply:
we use node 18 on build pipeline, so it's probably fair to stick to the same here